README.rst

```diff
@@ -133,66 +143,84 @@ For other build systems
+<https://github.com/xmos/lib_xcore_math/tree/master/example/>`_ directory:
 
-* `/test/ <https://github.com/xmos/lib_xcore_math/tree/develop/test/>`_ - Unit test projects for ``lib_xcore_math``.
+* `/example/ <https://github.com/xmos/lib_xcore_math/tree/master/example/>`_ - Example projects for ``lib_xcore_math``:
 
-  * `bfp_tests/ <https://github.com/xmos/lib_xcore_math/tree/develop/test/bfp_tests/>`_ - BFP unit tests 
-  * `vect_tests/ <https://github.com/xmos/lib_xcore_math/tree/develop/test/vect_tests/>`_ - Vector op unit tests
-  * `scalar_tests/ <https://github.com/xmos/lib_xcore_math/tree/develop/test/scalar_tests/>`_ - Scalar op unit tests
-  * `filter_tests/ <https://github.com/xmos/lib_xcore_math/tree/develop/test/filter_tests/>`_ - filtering unit tests
-  * `fft_tests/ <https://github.com/xmos/lib_xcore_math/tree/develop/test/fft_tests/>`_ - FFT unit tests
-  * `dct_tests/ <https://github.com/xmos/lib_xcore_math/tree/develop/test/dct_tests/>`_ - DCT unit tests
-  * `xs3_tests/ <https://github.com/xmos/lib_xcore_math/tree/develop/test/xs3_tests/>`_ - XS3-specific unit tests
+  * `bfp_demo/ <https://github.com/xmos/lib_xcore_math/tree/master/test/bfp_demo/>`_ - BFP demo
+  * `filter_demo/ <https://github.com/xmos/lib_xcore_math/tree/master/test/filter_demo/>`_ - Filtering demo
+  * `fft_demo/ <https://github.com/xmos/lib_xcore_math/tree/master/test/fft_demo/>`_ - FFT demo
+  * `vect_demo/ <https://github.com/xmos/lib_xcore_math/tree/master/test/vect_demo/>`_ - Vector op demo
 
-All unit tests are built and executed in a similar manner. The following shows how to do this with
+Each example above has a ReadMe file with a short description.
+
+
+All unit tests and examples are built and executed in a similar manner. The following shows how to do this with
 the BFP unit tests.
 
 BFP Unit Tests
 **************
 
 This application runs unit tests for the various 16- and 32-bit BFP vectorized arithmetic functions.
 This application is located at `/test/bfp_tests/
-<https://github.com/xmos/lib_xcore_math/tree/develop/test/bfp_tests>`_.
+<https://github.com/xmos/lib_xcore_math/tree/master/test/bfp_tests>`_.
 
-To execute the BFP unit tests on the explorer board, from your CMake build directory use the 
-following (after ensuring that the hardware is connected and drivers properly installed):
-
-::
+To execute the BFP unit tests on the explorer board, from your CMake build directory use the
+following (after ensuring that the hardware is connected and drivers properly installed): ::
 
     xrun --xscope test/bfp_tests/bfp_tests.xe
 
-Or, to run the unit tests in the software simulator:
-
-::
+Or, to run the unit tests in the software simulator: ::
 
     xsim test/bfp_tests/bfp_tests.xe
 
 Note that running the unit tests in the simulator may be *very* slow.
 
+To execute the BFP unit tests built for an x86 host platform, from your CMake build directory run on Linux and MacOS: ::
+
+   test/bfp_tests/bfp_tests -v
```

Comment:
It should be `. ./test/bin`

Reply:
no, the path in the text is correct.

Reply:
the command works for me on Ubuntu:

```
test/bfp_tests/bfp_tests 
.....................................................................................................................................
-----------------------
133 Tests 0 Failures 0 Ignored 
```

I used `./test/bfp_tests/bfp_tests` at the end and it is more common.

Reply:
fair